daprdocs/content/en/reference/api/metadata_api.md

```diff
@@ -6,9 +6,11 @@ description: "Detailed documentation on the Metadata API"
 weight: 1100
 ---
 
-Dapr has a metadata API that returns information about the sidecar allowing runtime discoverability. The metadata endpoint returns a list of the resources (components and HttpEndpoints loaded), the activated actors (if present), and attributes with information attached.
+Dapr has a metadata API that returns information about the sidecar allowing runtime discoverability. The metadata endpoint returns the runtime version, a list of the resources (components and HttpEndpoints loaded), the activated actors (if present), subscriptions, enabled features, app connection details, and attributes with information attached.
```

Comment:
This raises the issue, of the need to return a list of the resiliency policies as well, which is missing.

```suggestion
Dapr has a metadata API that returns information about the sidecar allowing runtime discoverability. The metadata endpoint returns the following information.
- Runtime version
- List of the loaded resources (components, subscriptions and HttpEndpoints)
- Activated actors (if present),
- Enabled features,
- Application connection details
- Custom, ephemeral attributes with information.

```

Reply:
@msfussell interesting idea, do you have any specific usecase in mind and how much details should the metadata API expose? Just the name of the registered policies? targets?